src/Umbraco.Web.UI.Client/src/packages/tiptap/components/input-tiptap/input-tiptap.element.ts

```diff
@@ -223,6 +231,7 @@ export class UmbInputTiptapElement extends UmbFormControlMixin<string, typeof Um
 		return html`
 			<umb-tiptap-toolbar
 				data-mark="tiptap-toolbar"
+				aria-label=${this.#getLabel('toolbar')}
```

Comment:
The toolbar element isn't an input itself, so shouldn't require an `aria-label` attribute. Each of the toolbar buttons/actions would have their own corresponding labels.